Packs/FeedMISPThreatActors/Integrations/FeedMISPThreatActors/README.md

```diff
@@ -0,0 +1,37 @@
+# MISP Threat Actors Feed Integration
+
+## Overview
+This integration fetches threat actor information from the MISP Threat Actors Galaxy and creates indicators in XSOAR. It provides valuable threat intelligence about various threat actors, including their aliases, targets, origin countries, and related information.
+
+## Use Cases
+- Fetch and update threat actor information regularly
+- Enrich your threat intelligence with detailed information about known threat actors
+- Create relationships between threat actors and their targets or aliases
+
+## Configuration
+1. Navigate to **Settings** > **Integrations** > **Servers & Services**.
```

Comment:
Navigate to **Settings & Info** > **Settings** > **Integrations** > **Instances**.